client/src/js/services/CashService.js

```diff
@@ -142,15 +142,21 @@ function CashService(Modal, Api, Exchange, Session, moment, $translate) {
       { field : 'defaultPeriod', displayName : 'TABLE.COLUMNS.PERIOD', ngFilter : 'translate' },
       { field : 'invoiceReference', displayName: 'FORM.LABELS.INVOICE'},
       { field : 'patientReference', displayName: 'FORM.LABELS.REFERENCE_PATIENT'},
-      { field : 'invoice_uuid', displayName: 'FORM.LABELS.INVOICE'}
+      { field : 'invoice_uuid', displayName: 'FORM.LABELS.INVOICE' },
     ];
 
     // returns columns from filters
     return columns.filter(function (column) {
+      var LIMIT_UUID_LENGTH = 6;
```

Comment:
:+1: This is a good change to make for scenarios where we need to display a UUID. In the upcoming refactor on filters we should be able to display the patient's name to get around this bad UX.